sdk/textanalytics/azure-ai-textanalytics/tests/test_recognize_pii_entities.py

```diff
@@ -573,4 +573,22 @@ def test_recognize_pii_entities_v3(self, client):
         with pytest.raises(NotImplementedError) as excinfo:
             client.recognize_pii_entities(["this should fail"])
 
-        assert "'recognize_pii_entities' endpoint is only available for API version v3.1-preview.1 and up" in str(excinfo.value)
\ No newline at end of file
+        assert "'recognize_pii_entities' endpoint is only available for API version v3.1-preview.1 and up" in str(excinfo.value)
+
+    # currently only have this as playback since the dev endpoint is unreliable
+    @pytest.mark.playback_test_only
+    @GlobalTextAnalyticsAccountPreparer()
+    @TextAnalyticsClientPreparer(client_kwargs={
+        "api_version": TextAnalyticsApiVersion.V3_1_PREVIEW_2,
+        "text_analytics_account_key": os.environ.get('AZURE_TEXT_ANALYTICS_KEY'),
+        "text_analytics_account": "https://cognitiveusw2dev.azure-api.net/"
```

Comment:
we should not explore endpoint here. Should use the env var, such as "AZURE_TEXT_ANALYTICS_ENDPOINT"

Reply:
The endpoint isn't really a secret since it's included in our recordings, so I didn't think it was necessary to create a secret to pass recorded tests